src/components/search-form/search-form.css

```diff
@@ -1,56 +1,74 @@
 @import '@folio/stripes-core/src/components/variables';
 
 .search-form-container {
-  margin: 1em 0 0.5em 0;
-  width: 100%;
+  padding: 1em;
 }
 
 .search-switcher {
-  float: left;
-  margin: 0 1em;
+  display: block;
+  margin: 0 auto 1em auto;
+  position: relative;
+  vertical-align: middle;
+  width: 100%;
 
   & a {
-    display: inline-block;
     border: 1px solid var(--primary);
-    border-width: 1px 1px 1px 0;
+    border-radius: 0.4em;
+    display: inline-block;
+    flex: 0 1 auto;
     padding: 0.36em 0.57em;
+    position: relative;
+    text-align: center;
+    user-select: none;
     text-decoration: none;
-    line-height: 1;
+    vertical-align: middle;
+    white-space: nowrap;
+    width: calc(100% / 3);
 
     &:first-child {
-      border-left-width: 1px;
-      border-radius: 0.4em 0 0 0.4em;
+      margin-left: 0;
+
+      &:not(:last-child) {
+        border-top-right-radius: 0;
+        border-bottom-right-radius: 0;
+      }
     }
 
-    &:last-child {
-      border-radius: 0 0.4em 0.4em 0;
+    &:last-child:not(:first-child) {
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
     }
 
     &.is-active {
       background-color: var(--primary);
       color: #fff;
     }
+
+    &:not(:first-child):not(:last-child) {
+      border-radius: 0;
+    }
+  }
+
+  & a + a {
+    margin-left: -1px;
   }
 }
 
 .search-input {
-  height: var(--controlHeight);
-  min-height: var(--controlHeight);
-  margin-bottom: var(--controlMarginBottom);
-  padding: 4px;
-  border: 1px solid var(--inputBorderColor);
+  box-sizing: border-box;
+  border: 2px solid #ccc;
+  border-radius: 2px;
+  font-size: 13px;
+  background-color: #fff;
+  background-position: 10px 10px;
+  background-repeat: no-repeat;
+  padding: 5px;
+  border-radius: 5px;
+  width: 100%;
 }
 
 .search-submit {
-  margin-left: 0.5em;
-  background-color: var(--primary);
-  border: 1px solid var(--primary);
-  border-radius: 0.4em;
-  color: #fff;
-
-  &:hover{
-    background-color: color(var(--primary) shade(8%));
-  }
+  display: none;
```

Comment:
Yea, this is def not accessible.

If it _really_ needs to be hidden, you should use some screen-reader friendly css:

```css
.visuallyhidden { 
  position: absolute; 
  overflow: hidden; 
  clip: rect(0 0 0 0); 
  height: 1px; 
  width: 1px; 
  margin: -1px; 
  padding: 0; 
  border: 0; 
}
```

But imho, this is not ideal. How would a mouse-jockey click search? Where's the indication that you have to press enter to perform the search?

Reply:
@wwilsman yeah, let's just kick the can down the road and stick with a solution that we _know_ works. I put the button back. I'll follow up with other FOLIO UX folks on what our standard will be. 

![localhost-3000-eholdings-search-packages iphone 5](https://user-images.githubusercontent.com/230597/30943369-2ea4fcec-a3b6-11e7-9f35-f95c8500b509.png)